plugin/mock_context.go

```diff
@@ -96,11 +99,16 @@ func (c *FakeContext) WithAddress(addr loom.Address) *FakeContext {
 	return clone
 }
 
-func (c *FakeContext) CreateContract(contract Contract) loom.Address {
-	addr := createAddress(c.address, c.contractNonce)
+func (c *FakeContext) CreateContract(contract Contract) (loom.Address, error) {
+	addr, err := createAddress(c.address, c.contractNonce)
+	if err != nil {
```

Comment:
Instead of returning the error `panic` instead, then we won't have to change a bunch of test code in `loomchain` for no good reason.